src/styles/blocks/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400px;
+  font-size: 10px;
+  color: #616070;
+
+  padding: 32px 16px 16px;
+}
+
+.card__image-imac {
+  display: block;
+  width: 160px;
+  height: 134px;
+
+  margin: 0 3px 40px ;
+}
+
+.card__name-of-product {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+
+  margin-bottom: 4px;
+}
+
+.card__code-of-product {
+  margin-bottom: 16px;
+
+  line-height: 14px;
+}
+
+.card__product-rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+}
+
+.rating__reviews {
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  margin-bottom: 16px;
+}
+
+.price__text {
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.price__value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: #060b35;
+}
+
+.card__buy-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  background-color: #00acdc;
+  border: 1px transparent #00acdc;
+  border-radius: 5px;
+
+  width: 166px;
+  height: 40px;
+
+  color: #fff;
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-transform: uppercase;
+
+  transition: all 0.3s;
```

Comment:
It is better to add specific properties to the transition so that the browser does not watch to all properties, e.g.: `transition: color 0.3s ease, background-color 0.3s ease;`